Apps/Playground/package.json

```diff
@@ -3,10 +3,51 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "select": "node scripts/version.js selectRN"
+    "android": "react-native run-android",
+    "build:android": "mkdirp dist/res && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res",
+    "build:ios": "mkdirp dist && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.ios.jsbundle --assets-dest dist",
+    "build:macos": "mkdirp dist && react-native bundle --entry-file index.js --platform macos --dev true --bundle-output dist/main.macos.jsbundle --assets-dest dist",
+    "build:windows": "mkdirp dist && react-native bundle --entry-file index.js --platform windows --dev true --bundle-output dist/main.windows.bundle --assets-dest dist",
+    "ios": "react-native run-ios",
+    "lint": "eslint .",
+    "macos": "react-native run-macos --scheme Playground",
+    "start": "react-native start",
+    "test": "jest",
+    "windows": "react-native run-windows --sln windows/Playground.sln"
+  },
+  "dependencies": {
+    "@babylonjs/core": "8.14.1",
+    "@babylonjs/loaders": "8.14.1",
+    "@babylonjs/react-native": "file:../../Modules/@babylonjs/react-native",
```

Comment:
Since the PackageTest directory and app are gone, can we test published packages (or locally built packages) by just installing that package into this project (temporarily overwriting the local source pakcage)?

Reply:
Yes, I can add a step after package build to test produced packed .tgz

Reply:
fixed